### PR TITLE
Baseline OETH analytics & Dune data integrations

### PR DESCRIPTION
**Pull request by smitch88**
#15 

Baseline implementation and date integration with Dune
- Adds Overview, Collateral, Protocol Revenue, and Strategies page

View in deployment by navigating to `/analytics`

Address design issues outlined in #75 
Address product issues outlined in #80 

**TODO:**
Add `/analytics` link into the CMS

### CODE REVIEW

**Comment by rafaelugolini** [comment]
You can access the analytics on `/analytics`

https://preview-oeth-sbmitchell-rdzso0.herokuapp.com/analytics

File: components/Gradient2Button.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { PropsWithChildren } from "react";
+import { twMerge } from "tailwind-merge";
+
+interface Gradient2ButtonProps {
+  onClick?: () => void;
+  className?: string;
+  outerDivClassName?: string;
+}
+
+const Gradient2Button = ({
```
Comment (HrikB) [inline components/Gradient2Button.tsx:10]: You could just use the `GradientButton`. It's actually the same thing as `Gradient2Button` from ousd.com but with the oeth.com gradient color.

File: utils/analytics.ts [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+        callback: function (val) {
+          return val === 0 ? null : this.getLabelForValue(Number(val));
+        },
+      },
+    },
+  },
+};
+
+export const aggregateCollateral = ({ collateral, allocation }) => {
+  console.log({
```
Comment (HrikB) [inline utils/analytics.ts:178]: Remove log

**Comment by rafaelugolini** [comment]
@smitch88 7 day APY not showing
image

**Comment by matt-bullock** [comment]
These look good

**Comment by smitch88** [comment]
Adjust dune query response for 7 day, [apy query](https://dune.com/queries/2611997).

**Comment by smitch88** [comment]
Addressed items outlined in #80

**Pull request merged**